src/webapp/js/container/chiefdoms-table.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import ReactTable from 'react-table';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+
+import 'react-table/react-table.css';
+
+import MobileTable from '../components/mobile-table';
+import {
+  hasAuthority, MANAGE_FACILITIES_AUTHORITY,
+} from '../utils/authorization';
+import { fetchChiefdoms } from '../actions/index';
+
+const COLUMNS = [
+  {
+    Header: 'Name',
+    accessor: 'name',
+  }, {
+    Header: 'Parent District',
+    accessor: 'parent',
+  },
+];
+
+class ChiefdomsTable extends Component {
```

Comment:
I think you can add some base table for all of the locations and pass columns and location list as a parameter to remove this duplicated code

Reply:
probably - will look into that